### DISCUSSION TITLE
Authentication Methods

### DISCUSSION THREAD

**Discussion by pstetinac** [pspete/psPAS#439] [Q&A]
Is it possible to set default authentication method for Cyberark by PowerShell??
![image](https://user-images.githubusercontent.com/119850010/205602429-b5dd098d-b49f-4ae8-b16b-d952105b0c50.png)
Enable  / Disable it's available with Set-PASAuthenticationMethod, but i would like to set the default method for authentication.
Is it possible?

**Comment by pspete** [comment]
Hi @psteitnac 

This is the documentation referenced for those commands:

- https://docs.cyberark.com/Product-Doc/OnlineHelp/PAS/Latest/en/Content/SDK/Update_Authentication_method.htm?TocPath=Developer%7CREST%20APIs%7CAuthentication%7CAuthentication%20methods%20configuration%7C_____2#Bodyparameters
- https://docs.cyberark.com/Product-Doc/OnlineHelp/PAS/Latest/en/Content/SDK/Authentication-Method-Add.htm?tocpath=Developer%7CREST%20APIs%7CAuthentication%7CAuthentication%20methods%20configuration%7C_____1#Bodyparameters

`DefaultMethod` is not listed as an available parameter.
